app/helpers/forever_style_guide/application_helper.rb

```diff
@@ -5,12 +5,16 @@ module ApplicationHelper
 
     def trademark(copy)
       case copy
-      when "Forever", "Forever Historian", "Forever Valet", "pixels2Pages", "Historian", "Forever Live!"
+      when "Forever"
+        copy.upcase
+      when "Forever", "Forever Historian", "Forever Valet", "Forever Live!", "Forever Print"
+        copy[0..6].upcase + copy[7..copy.length-1] + "™"
+      when "pixels2Pages", "Historian"
         copy += "™"
-      when "Forever Account", "Forever Guarantee", "Forever Guarantee Fund", "Forever Membership", "Forever Guarantee Fund", "Forever Members", "Forever Retreats", "Forever Ambassador", "Forever Ambassadors", "Forever Print"
-        copy[0..6] + "™" + copy[7..copy.length-1]
+      when "Forever Account", "Forever Guarantee", "Forever Guarantee Fund", "Forever Membership", "Forever Guarantee Fund", "Forever Members", "Forever Retreats", "Forever Ambassador", "Forever Ambassadors"
+        copy[0..6].upcase + "™" + copy[7..copy.length-1]
       when "Forever Artisan", "Artisan", "Forever Storage"
-        copy += "®"
+          copy[0..6].upcase + copy[7..copy.length-1] + "®"
```

Comment:
`copy[0..6].upcase` is used to handle the capitalization of Forever, so including 'Artisan' in here results in 'ARTISAN®'. We'll need a separate statement for 'Artisan' by itself to just add the ®, no upcasing needed.

Reply:
'Artisan' still needs removed from this line (it's still being capitalized right now because it runs these checks in the order you see them)

Reply:
oops -___-

Reply:
Also, indention on this and line 19 is off